codelists/api.py

```diff
@@ -77,6 +77,8 @@ def codelists_get(request, owner=None):
 
     records = []
 
+    include_user_codelists = "include-users" in request.GET
```

Comment:
The comment block above should be updated now that request.GET may contain "include-users" as well.